### PR TITLE
Load WgPort from the configuration file

### PR DESCRIPTION
**Pull request by mlsmaycon**
Added additional common blacklisted interfaces

### CODE REVIEW

**Comment by braginini** [comment]
One thing I miss here. How would the other side know about the non-default port?
https://github.com/netbirdio/netbird/blob/553a13588bea8ac9a70c5001379a6ef6ed68fbc2/client/internal/proxy/noproxy.go#L39

File: client/internal/engine.go [hunk 1]
```diff
@@ -388,15 +388,15 @@ func signalCandidate(candidate ice.Candidate, myKey wgtypes.Key, remoteKey wgtyp
 	return nil
 }
 
-func signalAuth(uFrag string, pwd string, myKey wgtypes.Key, remoteKey wgtypes.Key, s signal.Client, isAnswer bool) error {
+func signalAuth(uFrag string, pwd string, myKey wgtypes.Key, myPort int, remoteKey wgtypes.Key, s signal.Client, isAnswer bool) error {
```
Comment (braginini) [inline client/internal/engine.go]: I think that we should be more explicit in calling this field. 
E.g. WireGuard listen port or native port

File: signal/proto/signalexchange.proto [hunk 2]
```diff
@@ -49,4 +49,5 @@ message Body {
   }
   Type type = 1;
   string payload = 2;
+  uint32 port = 3;
```
Comment (braginini) [inline signal/proto/signalexchange.proto]: I think that we should be more explicit in calling this field.
E.g. WireGuard listen port or native port

**Review by braginini — APPROVED** [review]

File: client/internal/engine.go [hunk 3]
```diff
@@ -770,6 +771,12 @@ func (e *Engine) receiveSignalEvents() {
 				return fmt.Errorf("wrongly addressed message %s", msg.Key)
 			}
 
+			if msg.GetBody().GetPort() != 0 {
+				conf := conn.GetConf()
+				conf.ProxyConfig.WgPort = int(msg.GetBody().GetPort())
+				conn.UpdateConf(conf)
+			}
+
```
Comment (braginini) [inline client/internal/engine.go]: I would rather keep this logic inside the conn.
It is not a concern of the Engine to decide.

**Review by braginini — DISMISSED** [review]

**Comment by mlsmaycon** [comment]
LGTM

**Pull request merged**